packages/dso-toolkit/src/styles/components/_dso-select.scss

```diff
@@ -1,3 +1,5 @@
+@use "../../../../sources/src/styling/variables";
```

Comment:
Dit moet 

```
@use "@dso-toolkit/sources/src/styling/variables";
```

worden. Dit komt omdat deze package (`dso-toolkit`) ook door afnemers gecompileerd wordt (oa. het CMS). Door een truuk in path mappings werkt mijn suggestie zonder dat `@dso-toolkit/sources` wordt geinstalleerd.

#mand

Reply:
aangepast